net/mtr/Makefile

```diff
@@ -34,6 +34,11 @@ define Package/mtr
   URL:=https://www.bitwizard.nl/mtr/
 endef
 
+define Build/Configure
+	$(call Build/Configure/Default)
+	$(SED) 's|UNKNOWN|$(PKG_VERSION)|g' $(PKG_BUILD_DIR)/config.h
```

Comment:
The expression seems to be too wild to me.  My main concern is that unintended replacement may happen unnoticed on version bump.  How about making the macro name `PACKAGE_VERSION` also part of the expression?

Reply:
I mean something like `'s|PACKAGE_VERSION UNKNOWN|PACKAGE_VERSION "$(PKG_VERSION)"|'`

Or, `'/PACKAGE_VERSION UNKNOWN/d'` , then add `-DPACKAGE_VERSION='"$(PKG_VERSION)"'` to `EXTRA_CFLAGS`

Reply:
I don't think it will work. There are 3 `UNKNOWN`s in the config.h
```
/* Define to the full name and version of this package. */
#define PACKAGE_STRING "mtr UNKNOWN"
```
```
/* Define to the version of this package. */
#define PACKAGE_VERSION "UNKNOWN"
```
```
/* Version number of package */
#define VERSION "UNKNOWN"
```

Reply:
@neheb What's your opinion?

Reply:
Other possible options are

 - Touching `.tarball-version` after default preparation actions
 - Patching `configure.ac` to take version info from environment variable

Reply:
Please also remember to bump `PKG_VERSION`